src/Yasumi/Provider/Latvia.php

```diff
@@ -57,7 +57,7 @@ public function initialize()
         $this->addHoliday($this->christmasEve($this->year, $this->timezone, $this->locale, Holiday::TYPE_OFFICIAL));
         $this->addHoliday($this->christmasDay($this->year, $this->timezone, $this->locale));
         $this->addHoliday($this->secondChristmasDay($this->year, $this->timezone, $this->locale));
-        $this->addNewYearsEve();
+        $this->addHoliday($this->newYearsEve($this->year, $this->timezone, $this->locale));
```

Comment:
Shouldn't this change be excluded from this PR since it isn't related to Denmark?

Reply:
Both Denmark and Latvia (and many other countries) celebrate New Year's Eve, so I moved `newYearsEve ()` to `CommonHolidays` for easier reuse. So there is no reason to keep a duplicate implementation in the Latvia class.